osu.Game/Audio/Effects/Filter.cs

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using ManagedBass.Fx;
+using osu.Framework.Audio.Mixing;
+using osu.Framework.Bindables;
+using osu.Framework.Graphics;
+
+namespace osu.Game.Audio.Effects
+{
+    public class Filter : Component
+    {
+        public BQFType FilterType = BQFType.LowPass;
+        public float SweepCutoffStart = 2000;
+        public float SweepCutoffEnd = 150;
+        public float SweepDuration = 100;
+        public Easing SweepEasing = Easing.None;
+
+        public bool IsActive { get; private set; }
```

Comment:
You could probably make this a bindable and have the enable/disable logic run on changes of that bindable rather than have two explicit `Enable()`/`Disable()` methods - unless there is a specific reason to keep them separate?

Reply:
Probably not even required.